sentry_js/static/src/js/web_client.js

```diff
@@ -0,0 +1,27 @@
+odoo.define('sentry_js.web_client', function (require) {
+    'use strict';
+
+    var WebClient = require('web.WebClient');
+
+    WebClient.include({
+        //We install Raven after Odoo installs it's own window.onerror handler,
+        //otherwise ours (traceKitWindowOnError) is simply overwritten.
+        bind_events: function() {
+            this._super();
+            var sentry_configured = Boolean(odoo.sentry_config);
+            if (sentry_configured) {
+                Raven.config(odoo.sentry_config.dsn, {
```

Comment:
@kkarolis @taezaz,

maybe you know how to nicely wrap external JS libs in Odoo so that they could be nicely `require`'d as modules?

Reply:
Sorry, I don't know how it could be done, considering everything in raven.js is already global.

Reply:
```
  } else if (typeof define === 'function' && define.amd) {
    define([], f);
```
I guess this doesn't get executed, because odoo doesn't use requirejs, but something of it's own. Haven't checked really. If requirejs was used, then this condition should be satisfied and instead of being made global, Raven would be requirejs module.

Reply:
Indeed they are [rolling their own](https://github.com/odoo/odoo/blob/2d46dbcbfd122eaf1686623cf8cf319178e31109/addons/web/static/src/js/boot.js#L231) which is not compatible ☹️

Reply:
It seems they don't use require for external libs.